force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -56,8 +68,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
+        // need to ask instructors on how to optimize this code
         Integer largestNum = null;
-
+        if (num1 == num2){
+            largestNum = num1;
+            return largestNum; 
```

Comment:
You can move the return all the way at the end. If you are setting the variable, no need to return in each condition.